src/gwt/src/org/rstudio/studio/client/workbench/views/packages/Packages.java

```diff
@@ -1269,7 +1058,6 @@ public int compare(PackageInfo o1, PackageInfo o2)
       packratContext_ = newState.getPackratContext();
       view_.setProgress(false);
       setViewPackageList();
-      setViewActions(newState);
```

Comment:
Definitely not worth a lot of investment! If it turns out to be too involved, we could scale back by just eliminating the action list entirely from the IDE (which would let us remove a considerable amount of code on the server end, too)